versions.yaml

```diff
@@ -35,4 +35,22 @@ versions:
   - 4.4.5
   - '4.4'
   - '4'
```

Comment:
latest tag should be there also, as it is latest for `mongodb4` repo

Reply:
fixed